app/javascript/packages/document-capture/services/upload.tsx

```diff
@@ -45,17 +43,14 @@ export function toFormData(object) {
  *
  * @return {UploadFormEntryError}
```

Comment:
JSDoc type annotations are ignored in TypeScript files. We should remove them in favor of explicit TypeScript syntax. You did that here already for the parameter `uploadFieldError`, but we're missing the return type, as well as parameter/return types for some of the other functions in this file (e.g. `toFormData`).

```suggestion
```